### PR TITLE
fix/#95 padding input

### LINKED ISSUE #95: Change View Ticket Styles, Report and Home
### Explanation:
In the entry view of the ticket number and in the report view where there are three text fields and one respectively, these texts must be centered, they give the appearance of being misaligned, the dasher logo appears on the home page, it must be put off .

### Screenshots

image

### Checklist

- [ ] Align texts in ticket view.
- [ ]  Align texts in report view.

### References

(https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=407%3A46)

### PR DESCRIPTION
**Pull request by Eghinner**
### Description

Same padding left and right and reduce width to preserve.

fixes #95

screenshot-localhost_3000-2022 04 05-10_00_07

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### How to test it

Go to 'confirm/1'

### Checklist:

- [x] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [x] Doesn't have spelling or grammatical problems.
- [x] Doesn't have unnecessary comments or debugging code.
- [x] The branch is updated with the last dev branch changes.

### CODE REVIEW

File: src/components/FormInput.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import styled from '@emotion/styled';
 
 export const FormInput = styled.input`
-  width: 268px;
+  width: 248px;
   max-width: 100%;
```
Comment (Israel-Laguan) [inline src/components/FormInput.jsx:5]: @Katsu08 please verify, these seems weird, maybe they need to switch?

**Review by e-azocar — APPROVED** [review]

File: src/components/FormInput.jsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   height: 53px;
   border-radius: 7px;
   box-shadow: 2px 5px 6px rgb(0 0 0 / 0.3);
   background: var(--light-gray-1);
-  padding-left: 20px;
+  padding: 0 20px;
   margin: 10px;
   border: none;
   border: 1px solid var(--light-gray-0);
```
Comment (heliomar-pena) [inline src/components/FormInput.jsx]: Border 1px is overwriting border none, can you check what really is needed and remove the other?

File: src/components/FormInput.jsx [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import styled from '@emotion/styled';
 
 export const FormInput = styled.input`
-  width: 268px;
+  width: 248px;
   max-width: 100%;
   height: 53px;
   border-radius: 7px;
   box-shadow: 2px 5px 6px rgb(0 0 0 / 0.3);
```
Comment (heliomar-pena) [inline src/components/FormInput.jsx]: This RGB color is really working? It doesn't have "," between each "0", and it's declaring the opacity dividing the color between the opacity?

It doesn't should be some like:

```suggestion
  box-shadow: 2px 5px 6px rgba(0, 0, 0, 0.3);
```

instead?

Can you check please?

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
I think the porpose of the issue #95 is to align the text in the different inputs. Can you check the align in thats inputs?

Currently on the design, some inputs have text align to the left, for example the `/check/1` view always have align left, check design [here](https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=649%3A411)

But on the website it issue have text align center, can you check please?

Design:

image

Website:
image

File: src/components/FormInput.jsx (hunk 1 above)
Comment (heliomar-pena) [reply src/components/FormInput.jsx:5]: This size is not applying to the input correctly
image

File: src/components/FormInput.jsx (hunk 2 above)
Comment (Eghinner) [reply src/components/FormInput.jsx]: image

File: src/components/FormInput.jsx (hunk 3 above)
Comment (Eghinner) [reply src/components/FormInput.jsx]: done

**Review by heliomar-pena — APPROVED** [review]
Good work man! Thanks for help us to improve the code that you was not write

File: src/components/FormInput.jsx (hunk 3 above)
Comment (heliomar-pena) [reply src/components/FormInput.jsx]: Thanks u man!

File: src/components/FormInput.jsx (hunk 2 above)
Comment (heliomar-pena) [reply src/components/FormInput.jsx]: THanks

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Pull request merged**